.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v2
+    file: '/.gitlab-ci-template.yml'
+
+build:
+  extends: .build
+
+staging:
+  variables:
+    K8S_SECRET_APPLY_MIGRATIONS: true
```

Comment:
`APPLY_MIGRATIONS` should only be used for local development. A migrate command (`APP_MIGRATE_COMMAND`) is more appropriate when used in CI/CD setup.

See for example [kukkuu's CI config](https://github.com/City-of-Helsinki/kukkuu/blob/develop/.gitlab-ci.yml).

Reply:
Sure 👍🏼 Done in 4ef0f93